src/presenters/form.js

```diff
@@ -39,6 +39,7 @@ const props = [
   'lastSubmission',
   'reviewStates',
   'createdBy',
+  'entityRelated',
```

Comment:
Presenters have been removed, so this list of properties no longer exists, so you won't need to add `entityRelated` to it.